Remotion/Web/Development.WebTesting.IntegrationTests/ScreenshotTest.cs

```diff
@@ -724,5 +748,66 @@ private class DerivedScreenshotWebTreeViewNodeControlObject : ScreenshotWebTreeV
       {
       }
     }
+
+    private class EllipseScreenshotTransformation<T> : IScreenshotTransformation<T>
+    {
+      public int ZIndex { get; }
+      private int Width { get; }
+      private int Height { get; }
+      private Brush Brush { get; }
+      public Point Offset { get; }
+
+      public EllipseScreenshotTransformation (int width, int height, Brush brush, int zIndex = default, Point offset = default)
+      {
+        Width = width;
+        Height = height;
+        Brush = brush;
+        ZIndex = zIndex;
+        Offset = offset;
+      }
+
+      public ScreenshotTransformationContext<T> BeginApply (ScreenshotTransformationContext<T> context)
+      {
+        context.Graphics.FillEllipse (
+            Brush,
+            context.ResolvedElement.ElementBounds.X + Offset.X,
+            context.ResolvedElement.ElementBounds.Y + Offset.Y,
+            Width,
+            Height);
+
+        return context;
+      }
+
+      public void EndApply (ScreenshotTransformationContext<T> context)
+      {
+      }
+    }
+
+    private class RemoveYScreenshotTransformation<T> : IScreenshotTransformation<T>

```

Comment:
The name might indicate that it can remove stuff across the entire Y axis. Perhaps rename the class to `RemovePixelsFromBottomScreenshotTransformation` or something like that